src/sentry/tasks/integrations.py

```diff
@@ -154,3 +155,65 @@ def kick_off_status_syncs(project_id, group_id, **kwargs):
                 'external_issue_id': external_issue_id,
             }
         )
+
+# Particulariliry For VSTS
+# TODO(lb): I want to say do this every 6 hours
```

Comment:
You can use celery-beat to schedule a task every 6 hours. The celery-beat schedule is in `src/sentry/conf/server.py`.